src/tinypeel/Peeling/PeelingIO.py

```diff
@@ -66,47 +66,59 @@ def readInSeg(pedigree, fileName, start=None, stop=None):
 def writeOutParamaters(peelingInfo):
     args = InputOutput.args
 
-    np.savetxt(args.out + ".genoError", peelingInfo.genoError, fmt="%f")
-    np.savetxt(args.out + ".seqError", peelingInfo.seqError, fmt="%f")
-    # np.savetxt(args.out + ".trans", peelingInfo.transmissionRate, fmt = "%f")
-    np.savetxt(args.out + ".maf", peelingInfo.maf, fmt="%f")
+    np.savetxt(args.out_file + ".geno_error_prob.txt", peelingInfo.genoError, fmt="%f")
+    np.savetxt(args.out_file + ".seq_error_prob.txt", peelingInfo.seqError, fmt="%f")
+    np.savetxt(
+        args.out_file + ".rec_prob.txt", np.empty((1, 1)), fmt="%f"
+    )  # not be realized, just as a placeholder
+    # np.savetxt(args.out_file + ".trans", peelingInfo.transmissionRate, fmt = "%f")
```

Comment:
@AprilYUZhang do you know what exactly is in `peelingInfo.transmissionRate`? I wonder if this anyway connected to mutations? Open an issue and we can look at it later.

Reply:
```
def setupTransmission(length, peelingInfo):
    if peelingInfo.positions is None:
        localMap = np.linspace(0, 1, num=peelingInfo.nLoci, dtype=np.float32)
    else:
        localMap = (
            peelingInfo.positions / peelingInfo.positions[-1]
        )  # This should be sorted. Need to add in code to check.
    for i in range(peelingInfo.nLoci - 1):
        distance = localMap[i + 1] - localMap[i]
        distance = distance * length
        peelingInfo.transmissionRate[i] = distance
```
I found the code that calculates the distance between a locus and an adjacent locus. It seems to be used in the updateSeg(), but now it has been abandoned.
```
# def updateSeg(peelingInfo):
#     # Idea: Split the chromosome into ~10 blocks with slightly different starts/stops. Estimate at each one and then sum.
#     # Not sure if this is the best way, but probably will work okay.

#     distances = [
#         estDistanceFromBreaks(0, peelingInfo.nLoci - 1, nBreaks, peelingInfo)
#         for nBreaks in [5, 10, 20]
#     ]
#     distance = np.mean(distances[0:2])
#     print(distances, ":", distance)

#     setupTransmission(distance, peelingInfo)
```

Reply:
@AprilYUZhang the above help flag has still `-outputkey` mentioned, while it should be `out_id_order`